src/components/gui/gui.jsx

```diff
@@ -46,15 +46,19 @@ const GUIComponent = props => {
                         marginTop: 8
                     }}
                 >
-                    <TabSelected>
-                        <TabText>Scripts</TabText>
-                    </TabSelected>
-                    <Tab>
-                        <TabText>Costumes</TabText>
-                    </Tab>
-                    <Tab>
-                        <TabText>Sounds</TabText>
-                    </Tab>
+                    <Box
+                        width={250}
+                    >
+                        <TabSelected>
+                            <TabText>Scripts</TabText>
+                        </TabSelected>
```

Comment:
I'm not a React developer, let alone a scratch-gui dev, but wouldn't it make sense to have some kind of state and/or class to control whether a tab is selected or not, rather than a whole new tag? (Likely `Tab` and `TabSelected` will have nearly the same behaviour, aside from styling.)